cdap-ui/app/tracker/templates/preview.html

```diff
@@ -59,3 +59,10 @@
   ng-if="PreviewController.previewData.length === 0 && !PreviewController.loading">
   <p>No data</p>
```

Comment:
Would this show both `No data` and the `Preview is not available for this ...` message?